OpenRA.Game/FieldLoader.cs

```diff
@@ -223,8 +224,22 @@ public static object GetValue(string fieldName, Type fieldType, MiniYaml yaml, M
 			}
 			else if (fieldType == typeof(string))
 			{
-				if (field != null && MemberHasTranslateAttribute[field] && value != null)
-					return Regex.Replace(value, "@[^@]+@", m => Translate(m.Value.Substring(1, m.Value.Length - 2)), RegexOptions.Compiled);
```

Comment:
This change breaks the automatism we already have established in https://github.com/OpenRA/OpenRA/pull/5152.

Reply:
I like it better however so I suggest the utility command should be adapted.

Reply:
Making this depend on the widget hierachy instead of variable names would break compatibility/consistency with translated strings in the rule files or c#.

Reply:
I forgot about that. In that case, this needs to be undone.